src/integrations/prefect-azure/prefect_azure/container_instance.py

```diff
@@ -130,7 +128,7 @@ class ACRManagedIdentity(BaseModel):
     )
 
 
-class AzureContainerInstanceJobResult(InfrastructureResult):
+class AzureContainerInstanceJobResult:
```

Comment:
Is this class still used somewhere? I think it can be removed

Reply:
Yeah, the worker return type /type hints